pkg/kubelet/cm/deviceplugin/device_plugin_handler_test.go

```diff
@@ -238,6 +255,27 @@ func (a *activePodsStub) updateActivePods(newPods []*v1.Pod) {
 	a.activePods = newPods
 }
 
+func TestHandlerStart(t *testing.T) {
+	podsStub := activePodsStub{
+		activePods: []*v1.Pod{},
+	}
+	m, err := NewDevicePluginManagerTestStub()
+	monitorCallback := func(resourceName string, added, updated, deleted []pluginapi.Device) {}
+	testHandler := &HandlerImpl{
+		devicePluginManager:                m,
+		devicePluginManagerMonitorCallback: monitorCallback,
+		allDevices:                         make(map[string]sets.String),
+		allocatedDevices:                   make(map[string]sets.String),
+		podDevices:                         make(podDevices),
+	}
+	as := assert.New(t)
```

Comment:
Can we use the `require package` ?

Reply:
I am still not convinced that why we should be strict in using `require` over `assert`. As you on my another PR:
1.uniformity: If you see both have been used in all the source code.
2. Test would stop immediately with require:  IMO, that depends on individual preference. 
I would rather prefer to keep assert for now.